src/PanoViewer/PanoViewer.ts

```diff
@@ -335,7 +335,8 @@ class PanoViewer extends Component<PanoViewerEvent> {
         tileConfig: {
           flipHorizontal: false,
           rotation: 0
-        }
+        },
+        gap: 0
```

Comment:
Don't you need to add a description to the API?

Reply:
Yeah I forgot that, I'll add one.